intfsyncd/intfsync.cpp

```diff
@@ -47,6 +47,11 @@ void IntfSync::onMsg(int nlmsg_type, struct nl_object *obj)
         return;
 
     key = LinkCache::getInstance().ifindexToName(rtnl_addr_get_ifindex(addr));
+
+    /* Don't sync lo, eth0, and docker0 routes */
+    if (key == "lo" || key == "eth0" || key == "docker0")
+        return;
```

Comment:
why not pass them the swss and let the orch agent to figure out? In our design, the apps are independent probably not a good place to figure out the list of ports, lags and vlans during the runtime.

---
In reply to: [65829595](https://github.com/Azure/sonic-swss/pull/34#discussion_r65829595) [](ancestors = 65829595)